website/docs/stitch-combining-schemas.md

```diff
@@ -4,7 +4,7 @@ title: Combining schemas
 sidebar_label: Combining schemas
 ---
 
-Schema stitching (`@graphql-tools/stitch`) creates a single GraphQL gateway schema from multiple underlying GraphQL services. Unlike [schema merging](/docs/merge-schemas), which simply combines local schema instances, stitching builds a combined proxy layer that delegates requests through to underlying service APIs. Stitching is a comperable alternative to [Apollo Federation](https://www.apollographql.com/docs/federation/).
+Schema stitching (`@graphql-tools/stitch`) creates a single GraphQL gateway schema from multiple underlying GraphQL services. Unlike [schema merging](/docs/merge-schemas), which simply combines local schema instances, stitching builds a combined proxy layer that delegates requests through to underlying service APIs. As of GraphQL Tools v7, stitching is a comperable alternative to [Apollo Federation](https://www.apollographql.com/docs/federation/) with automated query planning, merged types, and declarative schema directives.
```

Comment:
Sp comparable

Reply:
oops. noted in response to that handbook thank you. Trying to make sure the lib identities its contemporary feature additions that justify it making this comparison. Lots of devs have history with the tool and therefore prejudices about just how "comparable" it is.